rust-bins/wallet-notes/README.md

```diff
@@ -676,6 +676,29 @@ The returned value is a JSON object with the following fields:
 An example input to this request is in the file [create_account_transaction-input.json](files/create_account_transaction-input.json).
 An example output to this request is in the file [create_account_transaction-output.json](files/create_account_transaction-output.json).
 
+### serialize_token_transfer_parameters
+Semantics: Converts the given parameters into a serialized base16 string that can be used to create an update smart contract transaction's payload.
+
+This function takes as input a NUL-terminated UTF8-encoded string. The string
+must be a valid JSON object with fields
+
+- `"from"` ... address of the sender account.
+
+- `"to"` ... address of the receiver account or contract. This should be a JSON object with two fields:
+  
+  - `"type"` ... The value should be "AddressAccount" or "AddressContract" depending on which type the receiver is.
+
+  - `"address"` ... If the receiver is an account, this should be the base58 encoding of the account's address. If it is a contract, this field should be a JSON object with the fields `index` and `subindex`, both numbers.
```

Comment:
I have changed this (at least temporarily) so that `to` has to be an account address. 

If it is a contract address then there must also be an entrypoint supplied. That is invoked on that contract. We can keep it just an account for this version, otherwise we need to add a JSON/Serde instance for the `Receiver` type in base.